<!--StartFragment-->


# **UNDERSTANDING RNA SEQUENCING**

RNA sequencing (RNA Seq) is a technique for studying the transcriptome that involves collecting all RNA molecules in a cell or tissue. Unlike DNA, which is generally static, RNA changes dynamically in response to illness, stress, or therapy.

By examining RNA, scientists can identify whether genes are active or quiet under particular settings(RNA Sequencing Data: Hitchhiker’s Guide to Expression Analysis \[PeerJ Preprints], n.d.)

In recent years, RNA-Seq has emerged as the preferred approach for analyzing dynamic transcriptomes, leveraging high throughput sequencing technology. 

RNA-Seq research has shown the transcriptome's size, concept, and complexity(Jazayeri et al., 2015)

Transcriptomics studies a living organism's transcriptome. The transcriptome is the whole collection of transcripts and their quantities for a certain developmental stage or physiological state, and it may be characterized as a comprehensive list of all kinds of RNA molecules, coding or non-coding, expressed in a specific cell, tissue, or organ.

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXdGYB2t3IRUH1H6bRWeQSbT2mnB-7WoEV6GL4HUHGXSYHmFp0OWz9bOPV4WaYBCi1jsXHjvnSk7aSOxzU88uQQcDQtLNpC_nW2ZkT5nFwtg0uyOxfarYwIjHtWXzKhLBy8cYiiV8FRcc7ryDkhTY9u8X4Fi?key=JTcZuQ9x6EOQiQ1exDB38g)

\
\



# **IMPORTANCE OF RNA SEQUENCING**

The advancement of RNA sequencing has propelled transcriptome analysis into a new era of greater efficiency and reduced cost. Fig [1](https://link.springer.com/article/10.1186/s13045-020-01005-x/figures/1) shows the timeline of RNA sequencing technology.

Several recent advances in RNA-seq technologies have resulted in a more thorough characterization of RNA transcripts. These advancements include better transcription start site mapping, strand-specific measurements, gene fusion detection, small RNA characterization, and detection of alternative splicing processes. (Ozsolak & Milos, 2011)


## **INTERSECTION OF RNA-SEQ AND MACHINE LEARNING.**

RNA-Seq data can reveal the same piece of RNA mapped to various locations on the genome, a phenomenon known as multiple-mapping reads. MMRs can skew gene expression measures, affecting all subsequent analyses, such as gene expression comparisons.

A machine learning technique called GeneQC (Gene Expression Quality Control) was created to address this issue. GeneQC assesses the reliability of gene expression data by examining several genomic and transcriptome characteristics.

It employs sophisticated algorithms to provide a more precise view of where the ambiguity in gene mapping sits. This enables researchers to trust the gene expression data they utilize and concentrate on high-quality data for future studies. GeneQC also contributes to the improvement of techniques for re-aligning readings to obtain more accurate gene expression estimations when reliability is poor(McDermaid et al., 2018)

\



# **REFERENCES**

- Berge, K. V. D., Hembach, K., Soneson, C., Tiberi, S., Clement, L., Love, M. I., Patro, R., & Robinson, M. (2018). _RNA sequencing data: Hitchhiker’s guide to expression analysis_ (No. e27283v1). PeerJ Inc.[ https://doi.org/10.7287/peerj.preprints.27283v1](https://doi.org/10.7287/peerj.preprints.27283v1)

* Jazayeri, S. M., Melgarejo Muñoz, L. M., & Romero, H. M. (2015). RNA-SEQ: A GLANCE AT TECHNOLOGIES AND METHODOLOGIES. _Acta Biológica Colombiana_, _20_(2), 23–35.[ https://doi.org/10.15446/abc.v20n2.43639](https://doi.org/10.15446/abc.v20n2.43639)

- Hong, M., Tao, S., Zhang, L., Diao, L.-T., Huang, X., Huang, S., Xie, S.-J., Xiao, Z.-D., & Zhang, H. (2020). RNA sequencing: New technologies and applications in cancer research. _Journal of Hematology & Oncology_, _13_(1), Article 1.[ https://doi.org/10.1186/s13045-020-01005-x](https://doi.org/10.1186/s13045-020-01005-x)

* McDermaid, A., Chen, X., Zhang, Y., Wang, C., Gu, S., Xie, J., & Ma, Q. (2018). A New Machine Learning-Based Framework for Mapping Uncertainty Analysis in RNA-Seq Read Alignment and Gene Expression Estimation. _Frontiers in Genetics_, _9_.[ https://doi.org/10.3389/fgene.2018.00313](https://doi.org/10.3389/fgene.2018.00313)

<!--EndFragment-->
